test/unit/mbyte_spec.lua

```diff
@@ -153,39 +137,33 @@ describe('mbyte', function()
       eq(0x007f, mbyte.utfc_ptr2char_len(to_string({0x7f, 0x7f, 0xcc, 0x80, 0x80}), pcc, 5))
       eq(0, pcc[0])
       -- No second UTF-8 character
-      local pcc = to_intp()
       eq(0x007f, mbyte.utfc_ptr2char_len(to_string({0x7f, 0xc2, 0xcc, 0x80, 0x80}), pcc, 5))
       eq(0, pcc[0])
 
       -- Combining character U+0300
-      local pcc = to_intp()
       eq(0x007f, mbyte.utfc_ptr2char_len(to_string({0x7f, 0xcc, 0x80, 0xcc}), pcc, 5))
       eq(0x0300, pcc[0])
       eq(0x0000, pcc[1])
 
       -- Combining characters U+0300 and U+0301
-      local pcc = to_intp()
```

Comment:
Ugh, you're right, I totally ignored that this is a cdata object! I'll clear that up, thanks!